geode-core/src/main/java/org/apache/geode/internal/net/SSLConfigurationFactory.java

```diff
@@ -82,7 +83,7 @@ public static SSLConfig getSSLConfigForComponent(
     return sslConfig;
   }
 
-  private synchronized void registeredSSLConfigForComponent(
+  private void registeredSSLConfigForComponent(
```

Comment:
Just curious, why is the synchronized removed here, but then added for clearSSLConfigForAllComponents()?  Are we only worried about protecting multiple concurrent clears?

Reply:
Given that the registeredSSLConfig member is now a ConcurrentHashMap it is not required to synchronize the puts (performed on the registeredSSLConfigForComponent method). The clear nevertheless was not protected against concurrent invocations and now it is.